### PR TITLE
Implement FactSchema.{list,get}

### PR DESCRIPTION
**Pull request by c-w**
[project-fortis#51](https://github.com/CatalystCode/project-fortis/issues/51)

### CODE REVIEW

File: src/resolvers-cassandra/Facts/queries.js [hunk 1]
```diff
@@ -1,17 +1,81 @@
...
+  return new Promise((resolve, reject) => {
+    if (!args || !args.tagFilter || !args.tagFilter.length) return reject('No tags specified to fetch');
+
+    const query = makeListQuery(args);
+    return cassandraConnector.executeQuery(query.query, query.params)
+    .then(rows => {
+      const facts = rows.map(cassandraRowToFact);
+
+      resolve({
+        facts: facts
```
Comment (jcjimenez) [inline src/resolvers-cassandra/Facts/queries.js:50]: Where's my runTime at? =)

**Review by jcjimenez — APPROVED** [review]
LGTM with minor question.

File: src/resolvers-cassandra/Facts/queries.js (hunk 1 above)
Comment (c-w) [reply src/resolvers-cassandra/Facts/queries.js:50]: This is a cross-cutting concern that we should implement once for all the resolvers (e.g. as a decorator).

Comment (jcjimenez) [reply src/resolvers-cassandra/Facts/queries.js:50]: Agreed!

**Review by jcjimenez — APPROVED** [review]
LGTM

**Pull request merged**